packages/app/hooks/useEntryPrice.ts

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react";
+import { getEntryPrice } from "app/api/rest/entryPrice";
+
+const savedPrice: Map<string, number> = new Map();
```

Comment:
we should use some kind of storage for that. i.ex. `async storage`.
Also, we should save some kind of expiration time for value. And refetch it after it expires.

Reply:
started using useSWR hook. it gives us a cache and expiration time.